src/main/resources/application.conf

```diff
@@ -334,6 +334,10 @@ mantis {
     # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-161.md
     eip161-block-number = "1000000000000000000"
 
+    # EIP-609 fork block number (ETH Only)
+    # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-609.md
+    byzantium-block-number = "4370000"
```

Comment:
generally by convention we treat `application.conf` as ETC config file, so ETH should be disabled here. We are doing that by assigining to each ETH fork big number like ` eip161-block-number = "1000000000000000000"`.
To enable them in ETH you need to create byzantium config entries in files:
https://github.com/input-output-hk/mantis/blob/phase/release2_0/src/universal/conf/ethereumHF.conf and
https://github.com/input-output-hk/mantis/blob/phase/release2_0/src/universal/conf/ropsten.conf